### PR TITLE
CS audit fixes

### PR DESCRIPTION
**Pull request by talbaneth**

### CODE REVIEW

File: src/test/DssProxyActionsCropper.t.sol [hunk 1]
```diff
@@ -4,7 +4,7 @@ import "ds-test/test.sol";
 
 import "../DssProxyActionsCropper.sol";
 import {Cropper, CropperImp} from "../Cropper.sol";
-import {CdpRegistry} from "../CdpRegistry.sol";
+import {CdpRegistry} from "lib/dss-cdp-registry/src/CdpRegistry.sol";
```
Comment (gbalabasquer) [inline src/test/DssProxyActionsCropper.t.sol:7]:
```suggestion
import {CdpRegistry} from "dss-cdp-registry/CdpRegistry.sol";
```

**Comment by talbaneth** [comment]
@gbalabasquer can we merge this?

**Review by gbalabasquer — APPROVED** [review]
LGTM

**Review by hexonaut — APPROVED** [review]
LGTM. I'm assuming the `quit()` was removed from the actions due to the indeterminism created by unwrapping the urns along with the need to `tack` to get your rewards (or `flee` to exit without).

**Comment by talbaneth** [comment]
> LGTM. I'm assuming the `quit()` was removed from the actions due to the indeterminism created by unwrapping the urns along with the need to `tack` to get your rewards (or `flee` to exit without).

As discussed `quit()` was removed since it's just a wrapper for a low level call, which we don't expect to be used directly.

**Pull request merged**